google-map.html

```diff
@@ -378,9 +378,23 @@
     },
 
     _updateMarkers: function() {
-      this.markers = Array.prototype.slice.call(
+      var newMarkers = Array.prototype.slice.call(
           Polymer.dom(this.$.markers).getDistributedNodes());
 
+      // do not recompute if markers have not been added or removed
+      if (newMarkers.length === this.markers.length) {
+        var added = newMarkers.filter( function(m) {
+            return this.markers && this.markers.indexOf(m) === -1;
```

Comment:
The goal is not to redo work if markers have not changed. Markers have not changed if:
- if lists are same length, and
- all new markers are found in the old list
Lists could be same length, but markers in the list could be different.